content/en/docs/running.md

```diff
@@ -44,6 +44,28 @@ By default, starting the container will attempt to load and/or build the Falco k
 docker run ... -e SYSDIG_SKIP_LOAD=1 ... falcosecurity/falco:{{< latest >}}
 ```
 
+## Running Falco in a Kind cluster
+
+The easiest way to run Falco on a [Kind](https://github.com/kubernetes-sigs/kind) cluster is as follows:
+
+1. Create a configuration file (e.g., `kind-config.yaml`) and add this content to the file:
```

Comment:
1. Create a configuration file. For example: `kind-config.yaml`
2. Add the following to the file: